geode-core/src/main/java/org/apache/geode/management/BackupStatus.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.management;
+
+import java.util.Map;
+import java.util.Set;
+
+import org.apache.geode.cache.persistence.PersistentID;
+import org.apache.geode.distributed.DistributedMember;
+import org.apache.geode.distributed.internal.DM;
+import org.apache.geode.internal.cache.BackupUtil;
+
+/**
+ * The status of a backup operation, returned by
+ * {@link BackupUtil#backupAllMembers(DM, java.io.File,java.io.File)}.
+ * 
+ * @since GemFire 6.5
+ */
+public interface BackupStatus {
+
+  /**
+   * Returns a map of disk stores that were successfully backed up. The key is an online distributed
+   * member. The value is the set of disk stores on that distributed member.
+   */
+  Map<DistributedMember, Set<PersistentID>> getBackedUpDiskStores();
```

Comment:
Does this work as an open data type? I'm worried that this might break in JConsole due to `DistributedMember` and `PersistentID` not being primitives or common data types. If an MXBean operation returns `BackupStats` then it might break. These might need to change to Strings.

Reply:
That class is not used with MXBean: the DiskBackupStatus is.